insights/parsers/tests/test_getconf_pagesize.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from insights.parsers.getconf_pagesize import GetconfPageSize
+from insights.tests import context_wrap
+from insights.parsers import ParseException
+
+GETCONFPAGESIZE1 = """
+4096
+""".strip()
+
+GETCONFPAGESIZE2 = """
+16384
+""".strip()
+
+def test_getconf_PAGESIZE():
+    result = GetconfPageSize(context_wrap(GETCONFPAGESIZE))
+    print(result)
+
+class Testgetconfpagesize():
+    def test_getconf_PAGESIZE1(self):
+        result = GetconfPageSize(context_wrap(GETCONFPAGESIZE1))
+        assert result.get("page_size") == 4096
+
+    def test_getconf_PAGESIZE2(self):
+        result = GetconfPageSize(context_wrap(GETCONFPAGESIZE2))
+        assert result.get("page_size") == 16384
```

Comment:
You should also add a test that the `ParseException` is raised when `len(content) != 1`.  See [this example](https://github.com/RedHatInsights/insights-core/blob/4ae338b4345191ded4c36852988eaaa1209b9e53/insights/parsers/tests/test_gluster_vol.py#L90).